docs/reference/workflows/ecs/concepts/system-update-order.md

```diff
@@ -1,40 +1,47 @@
 <%(TOC)%>
 
-# System update order
+# ECS: System update order
 
-_This document relates to the [ECS workflow]({{urlRoot}}/reference/workflows/overview)._
+The SpatialOS GDK for Unity (GDK) defines several update groups which run relative to [`PlayerLoop.Update`](https://docs.unity3d.com/ScriptReference/Experimental.LowLevel.PlayerLoop.html). Most of your systems should belong to one of these groups to ensure that the system is run in the right order relative to GDK receive and send systems.
 
-Unity provides attributes to define the [update order of systems (Unity documentation)](https://github.com/Unity-Technologies/EntityComponentSystemSamples/blob/master/Documentation/reference/workflows/ecs_in_detail.md#system-update-order). These attributes are: `UpdateInGroup`, `UpdateBefore` and `UpdateAfter`.
+## Update groups
 
-> You can only have one attribute of each type assigned to a system. If multiple are assigned they might override each other.
+The groups are executed in the following order:
+
+* `SpatialOSReceiveGroup` - This group contains all of the systems related to receiving and handling data from SpatialOS.
+  * `InternalSpatialOSReceiveGroup` - This is used by the `SpatialOSReceiveSystem`.
+  * `GameObjectInitializationGroup` - This group contains all systems used to link GameObjects to SpatialOS entities.
+  * `RequireLifecycleGroup` - This group contains all systems used to inject objects that were marked with `[Require]` in Monobehaviours.
+* `SpatialOSUpdateGroup` - **Most systems which use SpatialOS components should run in this group.**
+* `SpatialOSSendGroup` - This group contains all systems related to sending data to SpatialOS .
+  * `InternalSpatialOSSendGroup` - This group is used by the `SpatialOSSendSystem` to handle sending all replicated components to SpatialOS.
+  * `InternalSpatialOSCleanGroup` - This group is used by the `CleanReactiveComponentsSystem` to remove all [reactive]({{urlRoot}}/reference/workflows/ecs/interaction/reactive-components/overview) and [temporary components]({{urlRoot}}/reference/workflows/ecs/concepts/temporary-components) at the end of each update loop.
```

Comment:
that list looks outdated. e.g. `FixedUpdateSystemGroup`

Reply:
we mention fixed update at the bottom of the page - but probably should mention `FixedUpdateSystemGroup` there too